CorsixTH/Lua/game_ui.lua

```diff
@@ -785,15 +785,15 @@ function GameUI:playAnnouncement(name, priority, played_callback, played_callbac
   self.announcer:playAnnouncement(name, priority, played_callback, played_callback_delay)
 end
 
--- Get the configured mouse drag button
+-- Check whether the configured mouse drag button is being held down (true) or not (false).
```

Comment:
```suggestion
--! Check whether the configured mouse drag button is being held down (true) or not (false).
```
By starting with a`"--! ` sequence our luadoc tool picks it up and adds it to the generated htnl documentation.